.github/workflows/bin/setup_wheel_env_linux.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 
 # Install dependencies
-yum install -y libuuid-devel zlib-devel java-1.8.0-openjdk-devel graphviz
+yum install -y libuuid-devel zlib-devel java-1.8.0-openjdk-devel graphviz xorg-x11-server-Xvfb wget
```

Comment:
FWIW - You might want to call this script something like `setup_wheel_env_centos.sh` or `setup_wheel_env_linux_centos.sh` incase you ever end up also testing Debian / Ubuntu or other Linux versions.